### DISCUSSION TITLE
Lookup meeting notes

### DISCUSSION THREAD

**Discussion by VonSchnitzel** [dendronhq/dendron#3968] [Q&A]
I'm not sure if I'm having some configuration/setting issues or if I'm missing something else, but when using CTRL+L to bring up lookup, I can't find any of my meeting notes (those created via the 'Dendron: Create Meeting Note'). Any notes via the template.meet schema aren't coming up in the note lookup search.

I assume it should be possible to bring these up in the same list as other notes or is that wrong?

**Comment by wookiefriseur** [comment]
They show up for me. Try the command `Dendron: Reload Index` ( `Ctrl+Shift+P` for commands) and also `Developer: Reload Window`. You can also delete `dendron.meet.schema.yml` (it will be generated next time you create a meet note).

# Invalid Frontmatter

Dendron skips items with invalid IDs. Maybe that's what's happening to you. Open any of your meet files and look at the frontmatter.

It can look like this, for example:

```md
---
id: abcdefghijklm1234567890
title: My Title
desc: ''
updated: 1777777777777
created: 1777777777777
traitIds:
  - meetingNote
---

## The actual note
The actual note
```
If it looks completely different from this, you can try a repair:

1. `Dendron: Doctor` ( `Ctrl+Shift+P` for commands)
2. `fixFrontmatter`


# Invalid Filename

Try to avoid those strings in the path/name of your notes:

- `md` (dendron will think it's the filename extension and do funny stuff with the hierarchy)
- reserved JavaScript words like `nil` (don't remember which exactly, but had some UI issues with reserved JS words)
- things mentioned in [Restrictions](https://wiki.dendron.so/notes/v21pacjod0eqgdhb7zo7fvw/)

# Multiple vaults
Do you have multiple vaults? Then set the filters like in the screenshot, and create a new meeting:
![grafik](https://github.com/dendronhq/dendron/assets/8905319/848ce803-8727-4167-8d0a-94dd78a24359)

Check which vault it's inserted into


# Other Stuff

It could help to see your `dendron.yml` and the `templates.meet.md` (yeah, `templates` is correct, it does not use the default namespace `template` from `templateHierarchy`). Also can't hurt to have a look at the wiki entry [Meeting Note](https://wiki.dendron.so/notes/x1d18w46rxu34tksct3d4lh/). You can get help quicker in the Dendron Discord.

**Comment by VonSchnitzel** [comment]
Hey, thank you for getting back to me. I should have updated my comment but it eventually resolved itself after a restart. I think running the commands you posted would have worked as well.

The meeting notes experience is still inconsistent for me and I'm not sure why. For example, sometimes I can create a new meeting note using the normal meeting template, then the next day when I want to create one, it's creating a standard note instead and not using the meeting template. I'm not bothered to troubleshoot this further however as I'm sure it's just a funky setup on my end. 

Thanks again.
